play-services-core/src/main/java/org/microg/gms/gcm/UnregisterReceiver.java

```diff
@@ -8,18 +8,24 @@
 import java.util.List;
 
 import static android.content.Intent.ACTION_PACKAGE_REMOVED;
+import static android.content.Intent.ACTION_PACKAGE_DATA_CLEARED;
+import static android.content.Intent.ACTION_PACKAGE_FULLY_REMOVED;
 import static android.content.Intent.EXTRA_DATA_REMOVED;
+import static android.content.Intent.EXTRA_REPLACING;
 
 public class UnregisterReceiver extends BroadcastReceiver {
     private static final String TAG = "GmsGcmUnregisterRcvr";
 
     @Override
     public void onReceive(final Context context, Intent intent) {
         Log.d(TAG, "Package changed: " + intent);
-        if (ACTION_PACKAGE_REMOVED.contains(intent.getAction()) && intent.getBooleanExtra(EXTRA_DATA_REMOVED, false)) {
+        if ((ACTION_PACKAGE_REMOVED.contains(intent.getAction()) && intent.getBooleanExtra(EXTRA_DATA_REMOVED, false) &&
+                intent.getBooleanExtra(EXTRA_REPLACING, true)) ||
```

Comment:
Use `!intent.getBooleanExtra(EXTRA_REPLACING, false)` instead of `intent.getBooleanExtra(EXTRA_REPLACING, true)`

Reply:
Done